_raml/apps/orders/examples/put_orders_orderid_request.json

```diff
@@ -0,0 +1,20 @@
+{
+  "orderId": "5639C473-4265-5299-FDE2-D5809AB36284",
```

Comment:
please add all the attributes which can be changed here. e.g. also comment, shippedOn and so on
(adapt the response as well)